src/hypothesis/internal/charmap.py

```diff
@@ -106,6 +107,38 @@ def categories():
     return tuple(_categories)
 
 
+def as_general_categories(cats, name='cats'):
+    """Return a tuple of Unicode categories in a normalised order.
```

Comment:
It says it returns it in a normalised order, but this doesn't actually seem to be the case: It returns it in set order (which is essentially random)

Reply:
In `tuple(c for c in cs if c in out)`, `cs` is our sorted tuple of categories, so this both normalises the order and drops any unrecognised elements.  (the latter is preserving old behaviour, and issues a deprecation warning above)